firedrake/slate/preconditioners.py

```diff
@@ -142,7 +142,7 @@ def initialize(self, pc):
         Smat = self.S.petscmat
 
         # Nullspace for the multiplier problem
-        nullspace = create_schur_nullspace(P, K * Atilde.inv,
+        nullspace = create_schur_nullspace(P, -(K * K.T).inv * K * Atilde,
```

Comment:
The operator `K` exhibits a semi-orthogonality property (they're not square). Numerically `K * K.T` is producing a scalar multiple of the identity. If I omit the local tensor inversion in this expression, I get the same null space vectors. So perhaps it's safe to leave it as `-K * Atilde`?